pkg/assets/assets.go

```diff
@@ -27,6 +25,26 @@ type Asset struct {
 // Assets is a list of assets.
 type Assets []Asset
 
+// NewRecursiveAssets walks through a directory and render each file as asset.
+func NewRecursiveAssets(dir string, data interface{}) (Assets, error) {
```

Comment:
if there isn't another new function, `New` should do.  I don't think people should get an option against recursive.

Reply:
done